docs/reference/rest_api_statistics.md

```diff
@@ -0,0 +1,91 @@
+# Statistics
+
+Query for one or several statistics.
+
+## Get statistics
+
+`GET /api/stats`
+
+**Parameters**
+
+| Name     | Type   | Location | Description                                          |
+| -------- | ------ | -------- | ---------------------------------------------------- |
+| `stat`   | string | body     | Configured query to execute.                         |
```

Comment:
how should we denote that the top-level key is free to choose but the object needs a `stat` and `params` values?